File: 猜数游戏.py
import random
b=int(input("please input b:"))
k=False
a=random.randint(0,100)
for i in range(10):
    if a>b:
        print("Too small")
    if a<b:
        print("Too big")
    if a==b:
        print("you are right")
        k=True
if !k:
    print("you false,the number is:" + str(a))

File: 猜数游戏.pas
program c_k;
 var a,b,c:integer;
 begin
 randomize;
 a:=random(100);
 c:=0;
 repeat
 c:=c+1;
 write('please input b:');
 readln(b);
 if b>a then writeln('Too big')
 else if b<a then writeln('Too small')
 else if b=a tehn writeln('you are right');
 until (c=10) or (a=b);
 if c=10 then writeln('you false,the number is:',a);
 readln;readln;
 end.